addons/mail/tests/discuss/test_rtc.py

```diff
@@ -108,9 +107,8 @@ def test_10_start_call_in_chat_should_invite_all_members_to_call(self):
             [
                 (self.cr.dbname, 'discuss.channel', channel.id),  # update new session
                 (self.cr.dbname, 'discuss.channel', channel.id),  # message_post "started a live conference" (not asserted below)
-                (self.cr.dbname, 'res.partner', self.user_employee.partner_id.id),  # update of last interest (not asserted below)
-                (self.cr.dbname, 'res.partner', test_user.partner_id.id),  # update of last interest (not asserted below)
                 (self.cr.dbname, 'res.partner', test_user.partner_id.id),  # incoming invitation
+                (self.cr.dbname, 'discuss.channel', channel.id), # update of last interest (not asserted below)
                 (self.cr.dbname, 'discuss.channel', channel.id),  # update list of invitations
```

Comment:
I fail to see what makes using `discuss.channel` instead of `res.partner` put this notif order after channel invitations?

Reply:
order is irrelevant, I changed it by mistake when copying the diff

Reply:
ok :)